pyeasee/__main__.py

```diff
@@ -335,10 +335,14 @@ def str_fixed_length(myStr, length: int):
     return myStr
 
 
+def main():
+    asyncio.run(async_main())
+
+
 if __name__ == "__main__":
     import time
 
     s = time.perf_counter()
-    asyncio.run(main())
+    asyncio.run(async_main())
```

Comment:
Should this just be main()?

Reply:
Ok, sorry, now I understand what you mean. Perhaps :)

Reply:
Changed it.